File: features/list/src/main/java/com/vp/list/model/ListItem.java
package com.vp.list.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class ListItem  implements Serializable {
    @SerializedName("Title")
    private String title;
    @SerializedName("Year")
    private String year;
    @SerializedName("imdbID")
    private String imdbID;
    @SerializedName("Poster")
    private String poster;

    public String getTitle() {
        return title;
    }

    public String getYear() {
        return year;
    }

    public String getID() {
        return imdbID;
    }

    public String getPoster() {
        return poster;
    }
}

File: features/list/src/main/java/com/vp/list/model/ListItem.kt
package com.vp.list.model

import com.google.gson.annotations.SerializedName

class ListItem(
  @SerializedName("Title") val title: String,
  @SerializedName("Year") val year: String,
  val imdbID: String,
  @SerializedName("Poster") val poster: String
)